### DISCUSSION TITLE
what version to use with WS2812B?

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperSerialEsp8266#14] [General]
When using HyperSerial Esp8266, what version should I download to use with WS2812B?
No sure which one of these the WS2812B should fall under:
_APA102_
_SK6812_
_WS8201_
_WS821x_

**Comment by awawa-dev** [comment]
WS821x

https://github.com/awawa-dev/HyperSerialEsp8266/releases/download/v5.0.0.0/HyperSerialEsp8266.ino.d1_mini_WS821x_FIRSTLED_ENABLED.bin

**Reply by DariBer** [reply]
Thank you so much! :-)
